charts/events/templates/deployment.yaml

```diff
@@ -73,7 +73,8 @@ spec:
             httpGet:
               path: /healthz
               port: 5171
-            timeoutSeconds: 5
-            failureThreshold: 10
+            initialDelaySeconds: 30
+            timeoutSeconds: 10
```

Comment:
I don't think I've ever seen a timeout. 10 seconds sounds like an extremely long time to return a reply. Coupled with failure threshold we'd get an extremely long worst-case liveness failure.